gallery/gallery/lib/demos/material/list_demo.dart

```diff
@@ -22,6 +22,7 @@ class ListDemo extends StatelessWidget {
   Widget build(BuildContext context) {
```

Comment:
Does the full screen dialog demo still work?

Reply:
No it does not. Good catch. Let me fix this.

Reply:
I wrapped it in a Navigator now to fix it, will mean that Focus won't work on this page. 

See screenshot of working page:
<img width="2044" alt="Screenshot 2019-12-19 at 16 57 50" src="https://user-images.githubusercontent.com/1770678/71188230-de5cbf80-2280-11ea-8ba9-38d83a57f268.png">

Reply:
That's fine for now, until we can revisit focus. Thanks!